frontend/components/timeSeries/TimeSeries.js

```diff
@@ -18,9 +18,11 @@ const TimeSeries = () => {
                         controls
                         autoPlay
                         loop
-                        src={`data:audio/wav;base64, ${audioSample}`}
+                        src={`data:audio/wav;base64, ${audioSample.sound}`}
```

Comment:
When you refactor an object like this, it's worth renaming: `audioSample` made sense when the whole payload was a single sample, but now it's a dict containing an audio sample and an image. If you don't rename at a moment like this, gradually your variable names become more and more out of sync with their purpose in the code, and overtime the code in your project becomes difficult to read and maintain.